praisonai_tools/tools/langextract_tool.py

```
@@ -0,0 +1,271 @@
+            document_id = file_path.stem
+            
+            # Use extract method with file content
+            return self.extract(
+                text=text,
+                extractions=extractions,
+                document_id=document_id,
+                output_path=output_path,
+                auto_open=auto_open
+            )
+            
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)